packages/entities/entities-shared/src/types/entity-link.ts

```diff
@@ -0,0 +1,5 @@
+export interface EntityLinkData {
+  id: string,
+  label?: string,
+  deleted: boolean
```

Comment:
```suggestion
  deleted?: boolean
```

trivia: Maybe we can make this field optional?